src/mame/orca/akazukin.cpp

```diff
@@ -0,0 +1,618 @@
+	screen.set_palette(m_palette);
+
+	GFXDECODE(config, m_gfxdecode, m_palette, gfx_vastar);
+	PALETTE(config, m_palette, palette_device::RGB_444_PROMS, "proms", 256);
+
+	// sound hardware
+	SPEAKER(config, "mono").front_center();
+
+	AY8910(config, m_ay[0], XTAL(18'432'000) / 12);
+	m_ay[0]->port_a_read_callback().set_ioport("DSW1");
+	m_ay[0]->port_b_read_callback().set_constant(0xff);
+	m_ay[0]->add_route(ALL_OUTPUTS, "mono", 0.50);
+
+	AY8910(config, m_ay[1], XTAL(18'432'000) / 12);
+	m_ay[1]->port_a_read_callback().set_ioport("DSW2");
+	m_ay[1]->port_b_read_callback().set_constant(0xff);
+	m_ay[1]->add_route(ALL_OUTPUTS, "mono", 0.50);
+}
+
+
+ROM_START( akazukin )
+	ROM_REGION( 0x10000, "maincpu", 0 ) // on ORCA OVG-33c board
+	ROM_LOAD( "1.j2",  0x0000, 0x2000, CRC(8987f1e0) SHA1(a7a705f83a8b53c3c7b74b695a086516e3316219) )
+	ROM_LOAD( "2.l2",  0x2000, 0x2000, CRC(c8b040e4) SHA1(36b37f75080b682405ce1ad093538f6a1231fe72) )
+	ROM_LOAD( "3.m2",  0x4000, 0x2000, CRC(262edb0d) SHA1(afb2994274ae3d189ca3ffad556727afb0a8b1a4) )
+	ROM_LOAD( "4.n2",  0x6000, 0x2000, CRC(3569221f) SHA1(045cf99fe85b26503d368fecf1dad5f3785b5d79) )
+
+	ROM_REGION( 0x10000, "sub", 0 ) // on ORCA OVG-33c board
+	ROM_LOAD( "5.h7",  0x0000, 0x2000, CRC(c34486ae) SHA1(28b3db7e906202ee0d49907f66202544d352ff3f) )
+	ROM_LOAD( "6.j7",  0x2000, 0x1000, CRC(b5e1d77f) SHA1(637f17d04976bc43801012280010df8500313331) )
+
+	ROM_REGION( 0x1000, "fgtiles", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "10.b9",  0x0000, 0x1000, CRC(0145ded8) SHA1(515f3943579f2a8e17ca05959b9354981219d9d9) )
+
+	ROM_REGION( 0x4000, "sprites", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "9.e9",   0x0000, 0x2000, CRC(5fecc3d7) SHA1(619584936382c38c391c654684b24f6d642dff03) )
+	ROM_LOAD( "11.e7",  0x2000, 0x2000, CRC(448b28b8) SHA1(3a379c0a57ac89698b332d0af20ca8a3b0476d79) )
+
+	ROM_REGION( 0x1000, "bgtiles0", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "7.t4",  0x0000, 0x1000, CRC(7b4124da) SHA1(9bd713ac660920f6c5b407e80df9438df575de88) )
+
+	ROM_REGION( 0x2000, "bgtiles1", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "8.p4",  0x0000, 0x2000, CRC(193f6bcb) SHA1(0f4699052b2c66fabd293e7ef08fd25de7af42f3) )
+
+	ROM_REGION( 0x0300, "proms", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "r.r6",  0x0000, 0x0100, CRC(77ccc932) SHA1(5bd23ca5ab80ac9c19e85ea79ba3d276c4be59cb) ) // red component
+	ROM_LOAD( "g.m6",  0x0100, 0x0100, CRC(eddc3acf) SHA1(0fca5d36ccbd5191ce8a59d070112ae4a3297b0b) ) // green component
+	ROM_LOAD( "b.l6",  0x0200, 0x0100, CRC(059dae45) SHA1(26c9b975804fc206e80ee21361e489824f39e0c0) ) // blue component
+
+	ROM_REGION( 0x0100, "unkprom", 0 ) // on ORCA OVG-46C sub board
+	ROM_LOAD( "8n",  0x0000, 0x0100, CRC(e1b815ca) SHA1(df2b99259bf1023d37aa3b54247721d657d7e9c9) ) // ????
+ROM_END
+
+} // anonymous namespace
+
+
+// written as "Akazukin" on title screen
+GAME( 1983, akazukin, 0, akazukin, akazukin, akazukin_state, empty_init, ROT0, "Sigma", "Aka Zukin (Japan)", MACHINE_SUPPORTS_SAVE )
```

Comment:
I’d still write this as a single word “Akazukin” in the description as it’s being used like a Japanese given name, so it would be conventionally treated as a compound word.